acs-engine.go

```diff
@@ -107,6 +108,46 @@ func saveFile(dir string, file string, data []byte) error {
 	return nil
 }
 
+// injectCACertificateAndKey reads the ca cert and key files if they exist and inserts them into the container service model
+func injectCACertificateAndKey(containerService *api.ContainerService, caCertificatePath string, caKeyPath string) {
```

Comment:
for 2. i was trying to leave a comment to abstract a helper function for file path check, and retrieve data. Seems ioutil.Reafile already handled it.